board_orders.md

```diff
@@ -89,7 +89,9 @@ The overdue balance includes, but is not limited to, dues, fees, and merchandise
 **Article IV. Payment and Refunds**
 
 * Membership dues must be paid via a payment method sanctioned by the Treasurer.
-* Initiated members are not eligible for any form of refund.
+* If an initiated member who has already paid dues, decides before the Treasurer-specific dues date at midnight (CDT) that they for some reason no longer desire to be active, they are eligble for a full refund of what they have paid up till that date.

```

Comment:
Probably better worded as:

>Initiated members are eligible for a full refund of any paid membership dues less any third party processing fees until midnight (CDT) of the actives-specific dues date.

Reply:
Initiation happens at the end of the semester and it seems like after that point there should not be reimbursement after that point